packages/ketcher-core/src/domain/entities/sgroup.ts

```diff
@@ -632,7 +632,14 @@ export class SGroup {
     if (!sGroup) {
       return false
     }
-    return sGroup.type === 'SUP'
+    return sGroup.type === SGroup.TYPES.SUP
+  }
+
+  static isDataSGroup(sGroup) {
```

Comment:
I propose to add types here. 
And do we need this `if` condition?

Reply:
In a way this function replaces sgroup?.type === 'DAT', changed this to conditional chaining.